core/templates/dev/head/components/top_navigation_bar/TopNavigationBarDirective.js

```diff
@@ -43,13 +43,30 @@ oppia.directive('topNavigationBar', [
           $scope.NAV_MODE = GLOBALS.NAV_MODE;
           $scope.LABEL_FOR_CLEARING_FOCUS = LABEL_FOR_CLEARING_FOCUS;
           $scope.getStaticImageUrl = UrlInterpolationService.getStaticImageUrl;
-
+          
```

Comment:
Hi @srikar0896, the build is failing due to lint errors in this file. Here is the Travis log:  
```bash
/home/travis/build/oppia/oppia/core/templates/dev/head/components/top_navigation_bar/TopNavigationBarDirective.js
   46:1   error  Trailing spaces not allowed  no-trailing-spaces
  104:72  error  Trailing spaces not allowed  no-trailing-spaces
  116:71  error  Trailing spaces not allowed  no-trailing-spaces
  120:13  error  Trailing spaces not allowed  no-trailing-spaces
  127:32  error  Trailing spaces not allowed  no-trailing-spaces
  129:57  error  Trailing spaces not allowed  no-trailing-spaces
  133:66  error  Trailing spaces not allowed  no-trailing-spaces
  142:76  error  Trailing spaces not allowed  no-trailing-spaces
✖ 8 problems (8 errors, 0 warnings)
  8 errors, 0 warnings potentially fixable with the `--fix` option.
``` 
Please go through it once and fix these errors. Thanks!

Reply:
Done.